src/plone/app/mosaic/configure.zcml

```diff
@@ -1,9 +1,12 @@
 <configure
     xmlns="http://namespaces.zope.org/zope"
+    xmlns:five="http://namespaces.zope.org/five"
     xmlns:genericsetup="http://namespaces.zope.org/genericsetup"
     xmlns:zcml="http://namespaces.zope.org/zcml"
     i18n_domain="plone">
 
+  <five:registerPackage package="." />
```

Comment:
there is nothing to load in ``plone/app/mosaic/__init__.py`` so why did you add it?

Reply:
IIRC, without this the `Extensions` module is not loaded and there's no uninstall then; this is the case for Plone <= 5.0.

Reply:
Huh, Extensions. Is this still needed in 4.3? IIRC setuphandlers are fully supported in 4.3 latest?

Reply:
no, the uninstall profile doen't run automatically in Plone 4.3.